src/picongpu/include/traits/AdiosToPIC.tpp

```diff
@@ -44,6 +44,12 @@ namespace traits
         typedef unsigned int type;
     };
 
+    template<>
+    struct AdiosToPIC<adios_unsigned_long>
+    {
+        typedef uint64_t type;
```

Comment:
Offline discussed with many emotions :-)
Adios defines `adios_unsigned_long` as `unsigned long long` see [here](https://github.com/ornladios/ADIOS/blob/master/src/core/common_read.c#L1362-L1363)

We should do the same even if `unsigned long long` is not defined in `C++03` but in `C++11`.
`unsigned long long` is defined in `C99` and Adios is C code.

`unsigned long long` in C++11 is defined as `at least 64bit` see [here](http://de.cppreference.com/w/cpp/language/types)